src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/RespondToDefenceCallbackHandler.java

```diff
@@ -50,15 +53,15 @@
 
     private CallbackResponse handleNotifications(CallbackParams callbackParams) {
         Map<String, Object> data = callbackParams.getRequest().getCaseDetails().getData();
-        YesOrNo proceeding = mapper.convertValue(data.get(APPLICANT_1_PROCEEDING), YesOrNo.class);
-        var response = mapper.convertValue(data.get("respondent1ClaimResponseType"), DefendantResponseType.class);
-        if (response == FULL_DEFENCE && proceeding == YES) {
-            data.put("businessProcess",
-                     BusinessProcess.builder().activityId("CaseTransferredToLocalCourtHandling").status(READY).build());
+        CaseData caseData = mapper.convertValue(data, CaseData.class);
```

Comment:
Should we use CaseDetailsConverter here?

Reply:
done